src/site/content/en/fast/font-best-practices/index.md

```diff
@@ -178,9 +120,7 @@ connection](https://developer.mozilla.org/docs/Web/HTTP/CORS#what_requests_use_c
 </head>
 ```
 
-When using the `preconnect` resource hint, keep in mind that a font provider may
-serve stylesheets and fonts from separate origins. For example, this is how the
-`preconnect` resource hint would be used for Google Fonts.
+When using the `preconnect` resource hint, keep in mind that a font provider may serve stylesheets and fonts from separate origins. For example, this is how the `preconnect` resource hint would be used for Google Fonts.
```

Comment:
Ah, I see you do show the Google fonts snippet.

There's a lot of examples for `preconnect` here. Would it be possible to condense these and explain what it does after the code snippet? Might help for readability.